modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -3229,6 +3230,10 @@ private Integer findVarcharColumnLength(AttributeDescriptor att) {
                 sqlTypeNames[i] = sqlTypeName;
             }
         }
+        // Geos-7533 if all sql type names have been found in dialect dont
```

Comment:
Please refer to Geotools tickets, not geoserver ones

Reply:
changed to relevant GEOT, fixed